Dockerfile

```diff
@@ -1,23 +1,22 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
```

Comment:
Personally I'd pull the update of the Docker base image into it's own patch - given this might affect other things than just persist_to / replicateTo.

Reply:
Without this change docker image is not building, as gevent gives issues while installing on ubuntu 14.
If we skip this change, we have to lower gevent version to 1.2.0

Reply:
I'm not suggesting you remove it; just you upgrade to ubuntu 16.04 first (in a separate patch); and then add the persist_to / replicate_to changes.

Reply:
ok will do it